### DISCUSSION TITLE
How to use Mantine with NextJs 13 's app directory

### DISCUSSION THREAD

**Discussion by smyja** [mantinedev/mantine#4324] [Feature requests]
Is there any example on how to do this? What's available is for pages.

**Comment by TeshaneCrawford** [comment]
https://v7.mantine.dev/getting-started - Documentation of the upcoming version of Mantine UI
https://github.com/mantinedev/next-app-template/tree/master - Template you can use if you'd like to dive in

**Reply by smyja** [reply]
> https://v7.mantine.dev/getting-started - Documentation of the upcoming version of Mantine UI
> https://github.com/mantinedev/next-app-template/tree/master - Template you can use if you'd like to dive in

Thanks, this solves it.
